src/test/unit/sanitize.test.js

```diff
@@ -204,4 +206,76 @@ describe('sanitizePII', function() {
       expect(extensions.baseURL.length).toEqual(0);
     }
   });
+
+  it('should not sanitize all the preference values inside preference read markers', function() {
+    const profile = getProfileWithMarkers([
+      [
+        'PreferenceRead',
+        1,
+        {
+          type: 'PreferenceRead',
+          startTime: 0,
+          endTime: 1,
+          prefAccessTime: 0,
+          prefName: 'preferenceName',
+          prefKind: 'preferenceKind',
+          prefType: 'preferenceType',
+          prefValue: 'preferenceValue',
+        },
+      ],
+    ]);
+    const PIIToRemove = getRemoveProfileInformation({
+      shouldRemovePreferenceValues: false,
+    });
+
+    const sanitizedProfile = sanitizePII(profile, PIIToRemove).profile;
+    for (const thread of sanitizedProfile.threads) {
+      for (let i = 0; i < thread.markers.length; i++) {
+        const currentMarker = thread.markers.data[i];
+        if (
+          currentMarker &&
+          currentMarker.type &&
+          currentMarker.type === 'PreferenceRead'
+        ) {
+          expect(currentMarker.prefValue === 'preferenceValue').toBeTruthy();
```

Comment:
also, this is better:
```js
expect(currentMarker.prefValue).toEqual('preferenceValue');
```
This is better because when it fails the error message is much more useful.

Reply:
Another excellent suggestion. Thank you!